### ISSUE TITLE
error when requesting quartz/index

### ISSUE THREAD

**Issue by StefanTurbine** [jamescookie/quartz-monitor#34]
I use

```
    compile 'org.grails.plugins:quartz:2.0.4'
    compile 'org.grails.plugins:quartz-monitor:1.3'
```

on my Grails 3.1.1 (also tested with 3.0.17) build and get following Exception when requesting the monitor-pages:

`ERROR org.grails.web.errors.GrailsExceptionResolver - NullPointerException occurred when processing request: [GET] /quartz/list
Cannot get property 'jobGroupNames' on null object. Stacktrace follows:
java.lang.NullPointerException: Cannot get property 'jobGroupNames' on null object
    at grails.plugins.quartz.QuartzController$_closure2.doCall(QuartzController.groovy:22) ~[quartz-monitor-1.3.jar:na]
    at grails.plugin.springsecurity.web.filter.GrailsAnonymousAuthenticationFilter.doFilter(GrailsAnonymousAuthenticationFilter.groovy:53) ~[spring-security-core-3.0.4.jar:na]
    at grails.plugin.springsecurity.web.authentication.logout.MutableLogoutFilter.doFilter(MutableLogoutFilter.groovy:62) ~[spring-security-core-3.0.4.jar:na]
    at grails.plugin.springsecurity.web.SecurityRequestHolderFilter.doFilter(SecurityRequestHolderFilter.groovy:58) ~[spring-security-core-3.0.4.jar:na]
    at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142) ~[na:1.8.0_73]
    at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617) ~[na:1.8.0_73]
    at java.lang.Thread.run(Thread.java:745) [na:1.8.0_73]`

**Comment by jamescookie** [comment]
You're right, something was broken in 2.0.3 (fine in 2.0.2) and got fixed again by 2.0.8. Are you able to use a different quartz version?

**Comment by StefanTurbine** [comment]
There is another problem with the Quartz plugin that forced me to use this Version:
[click](https://github.com/grails-plugins/grails-quartz/issues/64)

I will try to downgrade to Version 2.0.2

EDIT:
Thank you, I works fine now with following versions.

```
    compile 'org.grails.plugins:quartz:2.0.2'
    compile 'org.grails.plugins:quartz-monitor:1.3'
```
